src/storage/addresses/addressesSelectors.ts

```diff
@@ -101,11 +103,22 @@ export const makeSelectAddressesAssets = () =>
           symbol: assetInfo?.symbol,
           description: assetInfo?.description,
           logoURI: assetInfo?.logoURI,
-          decimals: assetInfo?.decimals ?? 0
+          decimals: assetInfo?.decimals ?? 0,
+          verified: assetInfo?.verified
         }
       })
 
-      return [alphAsset, ...tokenAssets]
+      const [verifiedTokens, remainingTokens] = partition(tokenAssets, (token) => token.verified)
+      const [unverifiedTokens, unknownTokens] = partition(remainingTokens, (token) => token.verified === false)
+
+      return [
+        alphAsset,
+        ...(verifiedTokens as VerifiedAsset[]).sort((a, b) => a.name.toLowerCase().localeCompare(b.name.toLowerCase())),
+        ...(unverifiedTokens as UnverifiedAsset[]).sort((a, b) =>
+          a.name.toLowerCase().localeCompare(b.name.toLowerCase())
+        ),
+        ...unknownTokens.sort((a, b) => a.id.toLowerCase().localeCompare(b.id.toLowerCase()))
+      ]
```

Comment:
Nit: may be replaced by a one liner like (to be tested):

```ts
return sortBy(tokenAssets, [(a) => !a.verified, (a) => !symbol, (a) => a.name.toLowerCase()])
```

Reply:
wow, thanks for opening my eyes to `sortBy`! I discovered a rly need oneliner:

```ts
sortBy(assets, [(a) => !a.verified, (a) => a.verified === undefined, 'name', 'id'])
```

1. The first function brings the verified assets (`verified === true`) to the top of the list
2. The second function brings the `verified === false` assets right below the verified assets
3. The `'name'` argument sorts items where the `verified` prop is the same by `name`
4. The `'id'` argument sorts items where the `verified` and `name` props are the same by `id`

Fixed ✨

Reply:
You're welcome :) One thing: I don't know if the string sorting is actually case independent?

Reply:
Good point, but on second thought, I don't think we care a lot about case.

Reply:
Fixed! ✅